modules/claims_api/spec/requests/v1/power_of_attorney_request_spec.rb

```diff
@@ -142,6 +142,30 @@
       end
     end
 
+    context 'validate_veteran_identifiers' do
+      context 'when Veteran identifiers are missing in MPI lookups' do
+        before do
+          stub_mpi(build(:mvi_profile, birth_date: nil, participant_id: nil))
+        end
+
+        it 'returns an unprocessible entity status' do
+          allow(MPI::Messages::FindProfileMessageFields).to receive(:new).and_return(
+            OpenStruct.new({
+                             given_names: ['abraham'],
+                             last_name: 'lincoln',
+                             birth_date: nil,
+                             ssn: '796111863',
+                             missing_values: [:birth_date]
```

Comment:
there was a lot of things getting mocked through, so this was my quick way to get the error I wanted. let me know if there's a better, cleaner way to get to that same error in a test